packages/transport/src/types/messages.ts

```diff
@@ -1538,6 +1548,11 @@ export type Features = {
     busy?: boolean;
     homescreen_format?: HomescreenFormat;
     hide_passphrase_from_host?: boolean;
+    internal_model?: string;
+    unit_color?: number;
+    unit_btconly?: boolean;
```

Comment:
what is this @Hannsek ? btc_only firmware does not need to be guessed based on features.capabilities anymore?

Reply:
cc @matejcik

Reply:
oh yeah, I am reading comment in management proto 
```
unit/device is intended as bitcoin only
```
seems to be some hardwerdis btc-only. souds nice